### PR TITLE
feat(bux-705): Added gitbook

### PR DESCRIPTION
**Pull request by Nazarii-4chain**
# Pull Request Checklist

- [x] 📖 I created my PR using provided  : [CODE_STANDARDS](https://github.com/bitcoin-sv/spv-wallet/blob/main/.github/CODE_STANDARDS.md)
- [x] 📖 I have read the short Code of Conduct: [CODE_OF_CONDUCT](https://github.com/bitcoin-sv/spv-wallet/blob/main/.github/CODE_OF_CONDUCT.md)
- [ ] 🏠 I tested my changes locally.
- [ ] ✅ I have provided tests for my changes.
- [x] 📝 I have used conventional commits.
- [x] 📗 I have updated any related documentation.
- [x] 💾 PR was issued based on the Github or Jira issue.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,6 +15,8 @@
 [![standard-readme compliant](https://img.shields.io/badge/readme%20style-standard-brightgreen.svg?style=flat&v=3)](https://github.com/RichardLitt/standard-readme)
 [![Makefile Included](https://img.shields.io/badge/Makefile-Supported%20-brightgreen?=flat&logo=probot&v=3)](Makefile)
 <br/>
+
+[SPV Wallet GitBook](https://bsvblockchain.gitbook.io/docs)
```
Comment (chris-4chain) [inline README.md:19]: IMHO this link should be introduced with some text, e.g. what it is and what an user can find there.

File: README.md [hunk 2]
```diff
@@ -15,6 +15,11 @@
 [![standard-readme compliant](https://img.shields.io/badge/readme%20style-standard-brightgreen.svg?style=flat&v=3)](https://github.com/RichardLitt/standard-readme)
 [![Makefile Included](https://img.shields.io/badge/Makefile-Supported%20-brightgreen?=flat&logo=probot&v=3)](Makefile)
 <br/>
+
+## Documentation
+
+For in-depth information and guidance, please refer to the [SPV Wallet Documentation]([link here](https://bsvblockchain.gitbook.io/docs)).
+
```
Comment (chris-4chain) [inline README.md:22]: As @dorzepowski commented in the other repo:

> It actually should be somewhere below Table Of contents,

Comment (arkadiuszos4chain) [reply README.md:22]: I would put the sentence at the end of `About` paragraph:

```
More in-depth information and guidance about SPV Wallet Toolbox you can find [here](https://bsvblockchain.gitbook.io/docs).

Comment (Nazarii-4chain) [reply README.md:22]: Added sentence into `About` section.

Comment (arkadiuszos4chain) [reply README.md:22]: Please remove lines 18-22

Comment (Nazarii-4chain) [reply README.md]: Removed

**Review by arkadiuszos4chain — APPROVED** [review]

**Pull request merged**